101-servicebus-create-namespace-geo-recoveryconfiguration/azuredeploy.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
format the json, e.g. SHIFT+ALT+F in VS Code

Reply:
resolved